config/images/images.yaml

```diff
@@ -6,316 +6,320 @@ images:
   tags:
-  - v3.0.7
-  - v3.1.5
-  - v3.1.8
-  - v3.2.5
-  - v4.0.9
+  - source: v3.0.7
+  - source: v3.1.5
+  - source: v3.1.8
+  - source: v3.2.5
+  - source: v4.0.9
 # The kubesphere/fluent-operator image shall not be used anymore. Please use ghcr.io/fluent/fluent-operator/fluent-operator instead.
 - source: kubesphere/fluent-operator
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/kubesphere/fluent-operator
   tags:
-  - v1.7.0
-  - v2.2.0
-  - v2.3.0
-  - v2.7.0
+  - source: v1.7.0
+  - source: v2.2.0
+  - source: v2.3.0
+  - source: v2.7.0
 - source: ghcr.io/fluent/fluent-operator/fluent-operator
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/fluent-operator/fluent-operator
   tags:
-  - v2.9.0
-  - v3.1.0
-  - v3.2.0
-  - v3.3.0
+  - source: v2.9.0
+  - source: v3.1.0
+  - source: v3.2.0
+  - source: v3.3.0
 # DO NOT ADD NEW LOKI IMAGES.
 # With https://github.com/gardener/gardener/pull/7318 loki is replaced by vali.
 - source: grafana/loki
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/grafana/loki
   tags:
-  - 2.2.1
+  - source: 2.2.1
 # DO NOT ADD NEW PROMTAIL IMAGES.
 # With https://github.com/gardener/gardener/pull/7318 promtail is replaced by valitail.
 - source: grafana/promtail
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/grafana/promtail
   tags:
-  - 2.2.1
+  - source: 2.2.1
 # This source seems to be outdated. No new images are pushed here anymore.
 # Please use envoyproxy/envoy:distroless-<version> instead.
 - source: envoyproxy/envoy-distroless
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/envoyproxy/envoy-distroless
   tags:
-  - v1.24.1
-  - v1.26.4
-  - v1.26.7
-  - v1.29.2
-  - v1.29.3
-  - v1.30.1
-  - v1.30.2
-  - v1.30.3
-  - v1.30.4
-  - v1.31.0
-  - v1.31.1
-  - v1.31.2
-  - v1.32.0
-  - v1.32.1
-  - v1.32.2
-  - v1.32.3
-  - v1.33.0
-  - v1.33.1
-  - v1.33.2
-  - v1.34.0
-  - v1.34.1
-  - v1.34.2
-  - v1.34.3
-  - v1.35.0
+  - source: v1.24.1
+  - source: v1.26.4
+  - source: v1.26.7
+  - source: v1.29.2
+  - source: v1.29.3
+  - source: v1.30.1
+  - source: v1.30.2
+  - source: v1.30.3
+  - source: v1.30.4
+  - source: v1.31.0
+  - source: v1.31.1
+  - source: v1.31.2
+  - source: v1.32.0
+  - source: v1.32.1
+  - source: v1.32.2
+  - source: v1.32.3
+  - source: v1.33.0
+  - source: v1.33.1
+  - source: v1.33.2
+  - source: v1.34.0
+  - source: v1.34.1
+  - source: v1.34.2
+  - source: v1.34.3
+  - source: v1.35.0
 - source: envoyproxy/envoy
   destination: europe-docker.pkg.dev/gardener-project/releases/3rd/envoyproxy/envoy
   tags:
-  - distroless-v1.35.0
-  - distroless-v1.35.1
-  - distroless-v1.35.2
-  - distroless-v1.35.3
+  - source: distroless-v1.35.0
+    destination: v1.35.0-distroless
+  - source: distroless-v1.35.1
+    destination: v1.35.1-distroless
+  - source: distroless-v1.35.2
+    destination: v1.35.2-distroless
+  - source: distroless-v1.35.3
+    destination: v1.35.3-distroless
```

Comment:
I'm not sure if it we can make the update automation in renovate work with a reasonable effort ([ref](https://github.com/gardener/ci-infra/blob/59c7088bf6121ce39ede5e540248e40804f07732/.github/renovate.json5#L147-L160)) 🤔